### ISSUE TITLE
Scaledobjects of keda-add-ons-http-interceptor is not working

### ISSUE THREAD

**Issue by lambertpan** [kedacore/http-add-on#749]
### Report

I have recently updated keda and keda-http-add-on. The versions are below

- keda-2.11.2
- keda-add-ons-http-0.5.2
I see the following errors in keda-operator
2023-08-01T07:21:55Z	ERROR	scale_handler	error getting metric spec for the scaler	{"scaledObject.Namespace": "keda", "scaledObject.Name": "keda-add-ons-http-interceptor", "scaler": "externalScaler", "error": "got empty metric spec"}
In the scaled object status I see the following
 status:
    conditions:
    - message: Triggers defined in ScaledObject are not working correctly
      reason: TriggerError
      status: "False"
      type: Ready
    - message: Scaling is not performed because triggers are not active
      reason: ScalerNotActive
      status: "False"
      type: Active
    - status: Unknown
      type: Fallback
    - status: Unknown
      type: Paused
    hpaName: keda-hpa-keda-add-ons-http-interceptor
    originalReplicaCount: 3
    scaleTargetGVKR:
      group: apps
      kind: Deployment
      resource: deployments
      version: v1
    scaleTargetKind: apps/v1.Deployment

### Expected Behavior

scaled object of keda-add-ons-http-interceptor should work without an error.

### Actual Behavior

Error in scaled object of keda-add-ons-http-interceptor
status:
    conditions:
    - message: Triggers defined in ScaledObject are not working correctly
      reason: TriggerError
      status: "False"
      type: Ready
    - message: Scaling is not performed because triggers are not active
      reason: ScalerNotActive
      status: "False"
      type: Active
    - status: Unknown
      type: Fallback
    - status: Unknown
      type: Paused
    hpaName: keda-hpa-keda-add-ons-http-interceptor
    originalReplicaCount: 3
    scaleTargetGVKR:
      group: apps
      kind: Deployment
      resource: deployments
      version: v1
    scaleTargetKind: apps/v1.Deployment

### Steps to Reproduce the Problem

1. Install  keda-2.11.2 & keda-add-ons-http-0.5.2, Using hel chart
2. Check the logs of keda-operator
3.

### Logs from KEDA HTTP operator

```
2023-08-01T07:21:55Z	ERROR	external_scaler	error	{"type": "ScaledObject", "namespace": "keda", "name": "keda-add-ons-http-interceptor", "error": "rpc error: code = Unknown desc = httpscaledobject.http.keda.sh \"keda-add-ons-http-interceptor\" not found"}
github.com/kedacore/keda/v2/pkg/scalers.(*externalScaler).GetMetricSpecForScaling
	/workspace/pkg/scalers/external_scaler.go:172
github.com/kedacore/keda/v2/pkg/scaling/cache.(*ScalersCache).GetMetricSpecForScalingForScaler
	/workspace/pkg/scaling/cache/scalers_cache.go:104
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).getScaledObjectState
	/workspace/pkg/scaling/scale_handler.go:552
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).checkScalers
	/workspace/pkg/scaling/scale_handler.go:236
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).startScaleLoop
	/workspace/pkg/scaling/scale_handler.go:175
2023-08-01T07:21:55Z	ERROR	external_scaler	error	{"type": "ScaledObject", "namespace": "keda", "name": "keda-add-ons-http-interceptor", "error": "rpc error: code = Unknown desc = httpscaledobject.http.keda.sh \"keda-add-ons-http-interceptor\" not found"}
github.com/kedacore/keda/v2/pkg/scalers.(*externalScaler).GetMetricSpecForScaling
	/workspace/pkg/scalers/external_scaler.go:172
github.com/kedacore/keda/v2/pkg/scaling/cache.(*ScalersCache).GetMetricSpecForScalingForScaler
	/workspace/pkg/scaling/cache/scalers_cache.go:113
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).getScaledObjectState
	/workspace/pkg/scaling/scale_handler.go:552
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).checkScalers
	/workspace/pkg/scaling/scale_handler.go:236
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).startScaleLoop
	/workspace/pkg/scaling/scale_handler.go:175
2023-08-01T07:21:55Z	ERROR	scale_handler	error getting metric spec for the scaler	{"scaledObject.Namespace": "keda", "scaledObject.Name": "keda-add-ons-http-interceptor", "scaler": "externalScaler", "error": "got empty metric spec"}
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).getScaledObjectState
	/workspace/pkg/scaling/scale_handler.go:555
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).checkScalers
	/workspace/pkg/scaling/scale_handler.go:236
github.com/kedacore/keda/v2/pkg/scaling.(*scaleHandler).startScaleLoop
	/workspace/pkg/scaling/scale_handler.go:175
```

### HTTP Add-on Version

0.5.0

### Kubernetes Version

1.25

### Platform

Microsoft Azure

**Comment by t0rr3sp3dr0** [comment]
Hey @lambertpan, this is a duplicate of #718 and has been fixed by #720. Unfortunately, we still haven't released a new version of the add-on since the patch got merged. You can either build new images from the master branch of this repository and use them, or delete the ScaledObject for the Interceptor and scale the Interceptor manually.

**Comment by jacob-jameson** [comment]
Is it possible to release an image that contain this fix? My team wants to use this tool for an internal tool (we understand and accept the risks involved in the beta status of this add-on). We would like to not have to change the repo from which we pull the image, so a release containing this fix would be awesome.

**Comment by JorTurFer** [comment]
We build all the merged commits. You can use `canary` for using the main branch, or the commit sha for a specific commit (`6e7f15d` in this case).
I think that we could cut a release this month, WDYT @tomkerkhove @t0rr3sp3dr0 ?

**Comment by JorTurFer** [comment]
Hello,
The error was in the HTTP add-on code and chart. We have released today a new version (0.6.0) with this and other fixes, please give it a try and let me know if this has solved
